dell/latitude/3340/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  services.xserver.videoDrivers = lib.mkDefault [ "intel" ];
```

Comment:
By default many systems use `modsettings` driver for intel graphics because it is better maintained.
Are there specific reasons you are using intel here instead. Also see our 19.03 changelog:

> - The `intel` driver has been removed from the default list of [X.org video drivers](options.html#opt-services.xserver.videoDrivers). The `modesetting` driver should take over automatically, it is better maintained upstream and has less problems with advanced X11 features. This can lead to a change in the output names used by `xrandr`. Some performance regressions on some GPU models might happen. Some OpenCL and VA-API applications might also break (Beignet seems to provide OpenCL support with `modesetting` driver, too). Kernel mode setting API does not support backlight control, so `xbacklight` tool will not work; backlight level can be controlled directly via `/sys/` or with `brightnessctl`. Users who need this functionality more than multi-output XRandR are advised to add \`intel\` to \`videoDrivers\` and report an issue (or provide additional details in an existing one)

Reply:
ok i'll do some manual testing with my 3340 (which is rolling a basic gnome build) with this removed, to verify. 

The reason for line 10 here is that it's in the other latitudes' code.